server/errorHandler.test.ts

```diff
@@ -27,7 +27,7 @@ describe('GET 404', () => {
   it('should render content without stack in production mode', () => {
     const systemToken = async (user: string): Promise<string> => `${user}-token-1`
 
-    return request(appWithAllRoutes(null, null, systemToken, true))
+    return request(appWithAllRoutes(null, null, null, systemToken, true))
```

Comment:
Thinking at some point we change this to an object to make it easier to add stuff 🙂